e2e/testdata/fn-eval/resource-deletion/config.yaml

```diff
@@ -0,0 +1,21 @@
+# Copyright 2021 Google LLC
```

Comment:
Can we name this file to `starlark-delete-all.yaml` to make it obvious.

Reply:
change the file name to `starlark-delete-resource.yaml`